.travis.yml

```diff
@@ -22,11 +34,62 @@ matrix:
       env: CC=gcc
     - os: linux
       language: python
+      python: "3.3"
+      compiler: clang
+      env: CC=clang
+    - os: linux
+      language: python
+      python: "3.4"
+      compiler: gcc
+      env: CC=gcc      
+    - os: linux
+      language: python
       python: "3.4"
       compiler: clang
       env: CC=clang
     - os: linux
       language: python
+      python: "3.5"
+      compiler: gcc
+      env: CC=gcc      
+    - os: linux
+      language: python
+      python: "3.5"
+      compiler: clang
+      env: CC=clang
+    - os: linux
+      language: python
+      python: "pypy"
+      compiler: gcc
+      env: CC=gcc      
+    - os: linux
+      language: python
       python: "pypy"
       compiler: clang
       env: CC=clang
+    - os: linux
+      language: python
+      python: "pypy"
+      compiler: gcc
+      env: 
+        - CC=gcc
+        - PYENV_ROOT="$HOME/.pyenv"
+        - PATH="$PYENV_ROOT/bin:$PATH"
+      before_install:
+        - git clone https://github.com/yyuu/pyenv.git ~/.pyenv
```

Comment:
I don't like cloning and executing random versions of other repositories.  Also, it looks like we can expect to have PyPy 4 in Travis pretty soon.